python/GafferUI/SpreadsheetUI/_PlugTableView.py

```diff
@@ -648,13 +659,34 @@ def __getRowNameWidth( rowsPlug ) :
 		width = Gaffer.Metadata.value( rowsPlug.defaultRow(), "spreadsheet:rowNameWidth" )
 		return width if width is not None else GafferUI.PlugWidget.labelWidth()
 
-	def __editSelection( self ) :
+	def __editSelectedPlugs( self ) :
 
 		selectedPlugs = self.selectedPlugs()
 
+		if self.__mode == self.Mode.RowNames :
+			# Multi-editing row names makes no sense, so pick the first one.
+			# It will also be muddled up with the value cells.
+			firstRow = _PlugTableView.__firstRowPlug( selectedPlugs )
+			if not firstRow :
+				return
```

Comment:
How can this be? If we have a selection, then surely we should be able to find a row?

Reply:
Removed [here](https://github.com/GafferHQ/gaffer/pull/4001/commits/4f65b369a334a47c99b076e60b7ece4425df966c).

Reply:
Removed [here](https://github.com/GafferHQ/gaffer/pull/4001/commits/2542a19306ffb5b219e43273e9d4142e192479b6#diff-87e775b1c5b1cbc49fce36d38cc902ececd6e88e9ad9185757f2d763373858b5R685-R686).